src/main/java/online/book/store/service/imp/BookServiceImpl.java

```diff
@@ -31,9 +31,17 @@ public BookDto save(CreateBookRequestDto bookDto) {
         return bookMapper.toDto(savedBook);
     }
 
+    @Override
+    public BookDto update(CreateBookRequestDto bookDto, Long id) {
```

Comment:
It's good to check if this ID exists to throw an exception with proper message if not.